app/src/pages/EditPage/EditPage.tsx

```diff
@@ -0,0 +1,54 @@
+import React, {useState} from 'react';
+import {UserDataChange, UserPasswordChange, UserAvatarChange} from '../../components';
+import './EditPage.scss';
+
+enum ViewStatus {
+    DATA_EDIT,
+    PASSWORD_CHANGE,
+    AVATAR_CHANGE
+}
+export const EditPage = (): JSX.Element => {
+    const [view, setView] = useState(ViewStatus.DATA_EDIT);
+    let cnt: JSX.Element;
+    switch (view) {
+        case ViewStatus.PASSWORD_CHANGE:
+            cnt = <UserPasswordChange />;
+            break;
+        case ViewStatus.AVATAR_CHANGE:
+            cnt = <UserAvatarChange />;
+            break;
+        default:
+            cnt = <UserDataChange />;
+            break;
+    }
+
+    return (
+        <main className="ui">
+            <div className="ui__inner">
```

Comment:
сейчас существует UiLayout для этого
```suggestion
        <UiLayout>
```
Так же вероятно стоит установить высоту явно, так, чтобы при переключении вкладок, они(вкладки) оставались бы на одном уровне